AppKit/CPView.j

```diff
@@ -796,7 +796,7 @@ var CPViewFlags                     = { },
 
     do
     {
-        if (view == aView)
+        if (view === aView)
             return YES;
     } while(view = [view superview])
```

Comment:
ok, i see.
@ahankinson could you please insert a newline between brace and else?